.travis/upload.sh

```diff
@@ -0,0 +1,12 @@
+  #!/bin/bash
+
+set -e
+
+# Upload packages to hex
+# Only do so when on master
+
+if [[ ! -z "$TRAVIS_TAG" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+  mix hex.publish --organization simplifi --yes
```

Comment:
Not certain this sifi-specific bit is necessary, see [co_mix's `.travis/upload.sh`](https://github.com/simplifi/co_mix/blob/main/.travis/upload.sh)
```suggestion
  mix hex.publish --yes
```